Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ruby:2.3.1
+MAINTAINER Drew Bomhof (syncrou) https://github.com/syncrou
```

Comment:
[`MAINTAINER` is deprecated soon; use `LABEL` instead](https://github.com/docker/docker/blob/master/docs/deprecated.md#maintainer-in-dockerfile)